css/templates/lune/components/_header.css

```diff
@@ -0,0 +1,53 @@
+@import "../page-vars";
+
+.ampstart-headerbar {
+  justify-content: space-between
+
+  height: var(--layout-header-height-mobile);
+
+  /* Override ampstart default box shadow styles */
+  box-shadow: none;
+
+  /* header is fixed so should be moved onto a new GPU layer */
+  will-change: transform;
+}
+
+.ampstart-headerbar-home-link {
+  position: absolute;
```

Comment:
There is an existing way to do this. look at this template for example - https://ampstart.com/templates/alp.amp